controllers/machine_controller.go

```diff
@@ -313,3 +365,28 @@ func (r *MachineReconciler) Unlock(machine *naglfarv1.Machine, dockerClient *doc
 
 	return dockerClient.ContainerRemove(r.Ctx, container.ChaosDaemon, dockerTypes.ContainerRemoveOptions{Force: true})
 }
+
+func (r *MachineReconciler) deleteUploadDaemon(machine *naglfarv1.Machine, dockerClient *dockerutil.Client) error {
+	if err := dockerClient.PullImageByPolicy(container.MachineStatImage, naglfarv1.PullPolicyIfNotPresent); err != nil {
```

Comment:
What does the code act?

Reply:
![image](https://user-images.githubusercontent.com/59600278/105653946-f2e59f80-5ef7-11eb-8b8e-de1d0e82b0bb.png)
Emmm, this is an accident. It seems that Xixi here is redundant, too many copies

Reply:
done